### PR TITLE
Create configmap from versions.env file

### LINKED ISSUE #650: Create overrides config-map from versions.env file
AC:
- ConfigMap with overrides should be created based on versions.env file

### PR DESCRIPTION
**Pull request by strekm**
Resolves #650

### CODE REVIEW

File: installation/scripts/generate-local-config.ps1 [hunk 1]
```diff
@@ -3,6 +3,8 @@ $CURRENT_DIR = Split-Path $MyInvocation.MyCommand.Path
 $CONFIG_TPL_PATH = "${CURRENT_DIR}\..\resources\installer-config-local.yaml.tpl"
 $CONFIG_OUTPUT_PATH = (New-TemporaryFile).FullName
 
+$VERSIONS_FILE_PATH="${CURRENT_DIR}\..\versions.env"
```
Comment (crabtree) [inline installation/scripts/generate-local-config.ps1]: Missing spaces around =

File: installation/scripts/generate-local-config.ps1 [hunk 2]
```diff
@@ -24,6 +26,18 @@ Invoke-Expression -Command $cmd
 
 ##########
 
+Write-Output "Configuring versions ..."
+
+if([System.IO.File]::Exists($VERSIONS_FILE_PATH)){
+    $cmd = "kubectl create configmap versions --from-env-file=${VERSIONS_FILE_PATH} -n `"kyma-installer`""
```
Comment (crabtree) [inline installation/scripts/generate-local-config.ps1]: Please make the output of PS script similar to sh

File: installation/scripts/generate-local-config.ps1 (hunk 1 above)
Comment (strekm) [reply installation/scripts/generate-local-config.ps1]: done

File: installation/scripts/generate-local-config.ps1 (hunk 2 above)
Comment (strekm) [reply installation/scripts/generate-local-config.ps1]: in sh shouldn't be that echo instead

**Pull request merged**